kratos/python/add_strategies_to_python.cpp

```diff
@@ -371,8 +371,40 @@ namespace Kratos
             typedef BuilderAndSolver< SparseSpaceType, LocalSpaceType, LinearSolverType > BuilderAndSolverType;
 
 
-            py::class_< BuilderAndSolverType::DofsArrayType, BuilderAndSolverType::DofsArrayType::Pointer>(m,"DofsArrayType")
-                .def(py::init<>());
+            py::class_<BuilderAndSolverType::DofsArrayType, BuilderAndSolverType::DofsArrayType::Pointer>(m, "DofsArrayType")
```

Comment:
for whatever reason it formatted the file, so there were clashes all over

the only changes needed are these ones